### ISSUE TITLE
Keyb de / 850 is wrong 

### ISSUE THREAD

**Issue by blackcrack** [joncampbell123/dosbox-x#2637]
**Describe the bug**

the de Layout (gr)  on a quertz keyboard works not correctly
if you look :  
https://translate.google.com/translate?sl=auto&tl=en&u=https://de.wikipedia.org/wiki/Tastaturbelegung%23Besonderheiten_der_QWERTZ-Variante

https://en.wikipedia.org/wiki/German_keyboard_layout

is there a "ö_"/ "Ö_" and in the DosBox-X gives out " " and by .. "^°_"   (The _ is the blinking cursor )

**Expected behavior**

if i type "ö" should comes out a small "ö" and by shift a Big one so a "Ö"

**Screenshots**  
**Environment (please complete the following information):**
 - Operating system
 - DOSBox-X release version or commit SHA1 this issue occurred on
 - Used configuration, e.g. ```dosbox-x.conf```

![Screen 00003](https://user-images.githubusercontent.com/7925717/124376861-40391a80-dca9-11eb-81e3-08f2c6f3032a.jpg)

**Additional context**
Add any other context about the problem here.:
the Issue was already and was removed by dosbox-x-update-0.83.14-r2-20210507-x86.zip

![Screen 00004](https://user-images.githubusercontent.com/7925717/124376967-c0f81680-dca9-11eb-894c-b405110feb62.jpg)

**Comment by Wengier** [comment]
@blackcrack From the about screen you are using SDL1 build of DOSBox-X 0.83.15. It is known that the SDL version may sometimes affect the input keys for international keyboard. Can you try the SDL2 build of DOSBox-X 0.83.15 and see if it fixes the said problem? Thanks.

**Comment by blackcrack** [comment]
@Wengier , link ? because i have download all from the Gidhub-rel

do you meant this : dosbox-x-mingw-win32-20210701094904.zip ?

give me plz a link.. :) Thy :))))

**Comment by Wengier** [comment]
@blackcrack The simplest way is to just download the Windows installer, and select Visual Studio or MinGW SDL2 build from the installer selection. Alternatively, you can use portable binaries, and for example from the zip file you mentioned above, run the DOSBox-X binary in its mingw-sdl2 directory. Hope this helps.

**Comment by blackcrack** [comment]
@Wengier , you're right, it works, should input in the wiki, to know to have to use for Germans or other languages if not work the german,spain or geek keyboards, with key mutations to have take the sdl2 binary's ..

why works not in "normal" sdl1 binary and what' the decide between sdl1 and sdl2, this should writing in in the Wiki to have a possible for puttt innn an url if anyone comes with the same issues..

and also this with the 
Language =
title =
..
you could take directly the screenshot and make a wikientry with explain that .. ;)

best regards
Blacky

**Comment by Wengier** [comment]
@blackcrack Glad to hear that the problem is resolved with SDL2 builds. I think the build selection (SDL1 vs SDL2) belongs more to installation part rather than the usage part, so I added the following paragraph to the installation instructions:

``For most packages there are both SDL1 and SDL2 builds of DOSBox-X, and most features are the same for both builds. While SDL1 builds may be the default one to use, you may want to try SDL2 builds if you want certain features specific to SDL2 builds (such as the raw mouse input option) or you encounter specific issue(s) with SDL1 builds (such as incorrect keys in some international keyboard layouts).``

Thanks for the suggestion!

**Comment by maron2000** [comment]
Hi, you may want to check your mapper setting.
If it is not correctly set, neither SDL1 nor SDL2 version will work.

First, start from US keymode (`keyb us`), and type in order from left to right starting from `J`.
If it is not as shown in the screenshot below (`jkl;'\`) then open mapper tool and change the settings of respective keys.
Same can be applied to other rows as well.

My JP106 keyboard can switch between US and DE layout by `KEYB` command. (and of course JP layout) 

Hope it helps!

![de_keyb_result](https://user-images.githubusercontent.com/68574602/124403124-b738e680-dd6f-11eb-8b4a-8fb5c18ca3cf.png)
![mapper_tool](https://user-images.githubusercontent.com/68574602/124403380-0d5a5980-dd71-11eb-8b71-94effe2afe7c.png)

**Comment by blackcrack** [comment]
@maron2000 i did see the Kyb-Editor, but the issue is in the root.. -> Other Languages with mutated ?, use sdl2-folder or installer in the docu and a explain for that.. and also the explain over the other Gui/configuration possibility's with screenshot explains avout the single points .. then could the users easy go to the wiki and call the right page from the Configurationseditor and there is also directly explained what's the config in the dosbox-x.config file... this would be pretty nice... if someone would dare to approach
